tests/test_reviewers_only.py

```diff
@@ -96,10 +102,10 @@ def test_setup(self, openreview_client, helpers):
             ))
         
         helpers.await_queue_edit(openreview_client, edit_id=edit['id'])
-        helpers.await_queue_edit(openreview_client, invitation='openreview.net/Support/Simple_Dual_Anonymous/Venue_Configuration_Request/-/Submission_Template')
```

Comment:
can you add to this test the parentInvitations for submissions and reviews?

For example:
```
assert submissions[0].parent_invitations == 'openreview.net/Support/-/Submission_Template'
```

Reply:
Because `ABCD.cc/2025/Conference/-/Submission` is already in the note.invitations

Reply:
can you try querying the review by parent_invitations = openreview.net/Support/-/Review_Template?